cms/apps/media/admin.py

```diff
@@ -169,6 +253,34 @@ def response_add(self, request, obj, post_url_continue=None):
             return render(request, 'admin/media/file/filebrowser_add_success.html', context)
         return super().response_add(request, obj, post_url_continue=post_url_continue)
 
+    def change_view(self, request, object_id, form_url='', extra_context=None):
+        extra_context = extra_context or {}
+        # Strip all non-numeric characters from object_id to prevent an issue where object_id was returning a string (eg: '1231/change/none)
```

Comment:
This should never be the case. Django does not do this internally so you should not need to either.

Reply:
The page works fine but in my terminal I get 'ValueError: invalid literal for int() with base 10: '1231/change/none'' on page load.

Reply:
Resolved as I've reworked it such that it no longer overrides `change_view`.